.github/workflows/pull_request.yml

```diff
@@ -2,8 +2,8 @@ name: Pull Request
 
 on:
   pull_request:
-    types: [opened, reopened, synchronize, ready_for_review]
```

Comment:
This change will prevent running tests when a PR changes status from `draft` to `ready_for_review` I think.
Unless `synchronize` already covers that, which I don't think so.

Currently the tests won't be run if the PR is a `draft`.

Reply:
@MahdiBM they do run for draft PRs: https://github.com/apple/swift-nio/pull/2798

Reply:
hmmm that was not my experience with Vapor's CI which had this exact same setting.
I might have gotten something wrong then.
Thanks for confirming.

Reply:
I _think_ "opened" covers it. Draft / Ready for Review seems orthogonal to whether the PR is open or closed.

Reply:
Maybe then the case where it won't work is when you open the PR as draft, then when the CI is over, you add a commit, then after that commit, you mark the PR as `ready_for_review`.
_maybe_, not 100% sure.